content/en/docs/How To/create_output.md

```diff
@@ -0,0 +1,122 @@
+	Name:   "MyOutput",
+	New:    NewMyOutput,
+    Config: &MyOutputConfig{},
+    Raw:    true,
+	Help:   "Help message for the output",
+}
+```
+
+This object has a `Name`, that is used in the Baker configuration file to identify the output,
+a constructor-like function (`New`), a config object (used to parse the output configuration in the
+TOML file) and a help text that must help the users to use the component and its configuration
+parameters. The `Raw` field tells whether the output expects to receive raw records in addition
+to single fields (see below for details).
+
+### Output constructor-like function
+
+The `New` key in the `OutputDesc` object represents the constructor-like function.
+
+The function receives an [OutputParams](https://pkg.go.dev/github.com/AdRoll/baker#OutputParams)
+object and returns an instance of [Output](https://pkg.go.dev/github.com/AdRoll/baker#Output).
+
+`OutputParams` contains the index of the output and the fields `FieldIndex` that the output will
+receive during processing, in addition to the fields inherited from
+[ComponentParams](https://pkg.go.dev/github.com/AdRoll/baker#ComponentParams).
+
+`OutputParams.Index` indicates a unique index of the output process among the concurrent output
+processes generated by Baker. The `procs` configuration can be used to tune the total number of
+concurrent processes, see [Pipeline configuration](/docs/how-to/pipeline_configuration/) for details.  
+Note that the output should extensively document in `OutputDesc.Help` if it is able to manage
+concurrent processing or the user should configure it with a single process (`procs=1`).
+Read [Tuning concurrency](/docs/how-to/concurrency/) for an in-depth study on the subject.
+
+`OutputParams.Fields` is a list of [FieldIndex](https://pkg.go.dev/github.com/AdRoll/baker#FieldIndex)
+that the output will receive. Their position in this slice correspond to the fields in
+[OutputRecord.Fields](https://pkg.go.dev/github.com/AdRoll/baker#OutputRecord), see below for
+details.  
+If, for any reason, the output needs to retrieve the fields name (like the SQLite output does to
+get the columns names), then `OutputParams.FieldName` can be used.
+
+### The output configuration and help
+
+The output configuration object (`MyOutputConfig` in the previous example) must export all
+configuration parameters that the user can set in the TOML topology file.
+
+Each key in the struct must include a `help` string tag and a `required` boolean tag, both are
+mandatory.
+
+The former helps the user to understand the possible values of the field, the latter tells Baker
+whether to refuse a missing configuration param.
+
+## OutputRecord
+
+The [OutputRecord](https://pkg.go.dev/github.com/AdRoll/baker#OutputRecord) channel received by
+the output component is closed by Baker when all the records have been processed (in case of a batch
+execution) or when an interruption starts (in case of a SIGINT signal from the user).
```

Comment:
I'm not sure we should write about interruption (since an output doesn't directly deal with that), neither about signal here (it' os-dependent).
IMHO when writing an output, an user shouldn't have to consider these, their horizon should stop at the limit of Output interface.
The only _contract_ the user should respect however is that they should return from `Run` as soon as possible once the channel has been closed, whatever the reason is (be it that there are no more records to process or the topology is terminating prematurely).

Reply:
```The [OutputRecord](https://pkg.go.dev/github.com/AdRoll/baker#OutputRecord) channel received by
the output component can be closed by Baker at any time and the output should return from the `Run`
function as soon as possible when this happens.```

Reply:
:+1: